lib/tasks/rh_cloud_inventory.rake

```diff
@@ -2,6 +2,21 @@ require 'tempfile'
 
 namespace :rh_cloud_inventory do
   namespace :report do
+    desc 'Generate inventory report and send it to Red Hat cloud'
+    task generate_upload: [:environment, 'dynflow:client'] do
```

Comment:
I took `'dynflow:client'` part from https://github.com/theforeman/foreman-tasks/commit/81bf0968328e8eb3f921729eb42d9bdcb66c0a3a . Without it, trying to run task would result in exception:
```
rake aborted!
The Dynflow world was not initialized yet. If your plugin uses it, make sure to call Rails.application.dynflow.require! in some initializer
/opt/theforeman/tfm/root/usr/share/gems/gems/dynflow-1.4.7/lib/dynflow/rails.rb:75:in `world'
```

I don't understand why these two tasks require that, but existing tasks calling `execute_now` do not.

Reply:
Ah, I see it now - I might not use `perform_later` myself, but `GenerateReportJob` will schedule next step in commands chain using `perform_later`. Makes sense, thanks for explanation.